recipes/tomtoolkit/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "tomtoolkit" %}
+{% set version = "2.23.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/tomtoolkit-{{ version }}.tar.gz
+  sha256: 8a4364a31f4d7e3972509a8528808e7c4d6dda8e5a5743f6f1b834495bb65845
+
+build:
+  skip: true  # [py>=313 or py2k]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - poetry-core >=2.0,<3.0
+    - poetry-dynamic-versioning >=1.0.0,<2.0.0
+    - pip
+  run:
+    - python
+    - numpy >1.24,<2.2
+    - astroquery <1
+    - astroplan >=0.8,<0.11
+    - astropy >=5.3.3,<7  # [py>=310]
+    - astropy >=5.3.3,<6  # [py<310]
```

Comment:
Why does this package needs this constraint? Without it we can make it noarch.

Reply:
removed